uber/site_sections/preregistration.py

```diff
@@ -215,7 +215,7 @@ def prereg_payment(self, session, payment_id, stripeToken):
         elif charge.amount != charge.total_cost:
             message = 'Our preregistration price has gone up; please fill out the payment form again at the higher price'
         else:
-            message = charge.charge_cc(stripeToken)
+            message = charge.charge_cc(session, stripeToken)
```

Comment:
Hmm, interesting dilemma.  I think you're right that the ``Charge`` class is a better place for the ``charge_cc`` method than the ``Session`` class, so I think the way you have it here is probably the best approach.